boards/arm/rcar_h3ulcb/rcar_h3ulcb_cr7.yaml

```diff
@@ -0,0 +1,8 @@
+identifier: rcar_h3ulcb_cr7
+name: Cortex r7 for Renesas H3ULCB
+type: mcu
+arch: arm
+toolchain:
+  - zephyr
+  - gnuarmemb
+ram: 768
```

Comment:
How about flash @julien-massot ?

Reply:
There is no flash support at the moment. The cortex-r7 is a remote processor of the CA57/CA53, we only use ram area at the moment, the CA57/CA53 processors load the firmware and then start this CR7 processor.